fplus-http-server/src/router/application.rs

```diff
@@ -546,7 +585,10 @@ pub async fn request_kyc(query: web::Query<VerifierActionsQueryParams>) -> impl
         .request_kyc(&query.id, &query.owner, &query.repo)
         .await
     {
-        Ok(()) => HttpResponse::Ok().body(serde_json::to_string_pretty("Success").unwrap()),
+        Ok(_) => match serde_json::to_string_pretty("Success") {
```

Comment:
to_string_pretty won't fail if passed a static string. It's a good candidate for .expect (won't repeat for any others, please review all these cases).